arwen/host/execution.go

```diff
@@ -18,7 +18,10 @@ func (host *vmHost) doRunSmartContractCreate(input *vmcommon.ContractCreateInput
 	var err error
 	defer func() {
 		if err != nil {
-			message := err.Error() + " - " + output.ReturnMessage()
+			message := output.ReturnMessage()
+			if err != arwen.ErrSignalError {
```

Comment:
`If is errSignalError, then ... else ...` may be a bit readable because of the `else`. Just a suggestion.

Reply:
Changed.